### ISSUE TITLE
Command CompileSwiftSources failed with a nonzero exit code

### ISSUE THREAD

**Issue by michaelbutler1998** [capacitor-community/sqlite#225]
**Describe the bug**
An error upon runtime of "Command CompileSwiftSources failed with a nonzero exit code"

**To Reproduce**
1. Upgrade from Capacitor 2 to Capacitor 3
2. uninstall & reinstall the plugin
`"npm uninstall @capacitor-community/sqlite && npm install @capacitor-community/sqlite" `
3. Import into the file where used:` import { SplashScreen } from "@capacitor/splash-screen";`
4. Run in XCode

**Expected behavior**
No error upon runtime

<img width="1787" alt="image" src="https://user-images.githubusercontent.com/15836316/154870053-282f1a4f-eb38-4130-baf7-8c0b2e48afb5.png">

**Comment by jepiqueau** [comment]
@michaelbutler1998 Can you share your package.json file. after the step 2 did you remove the ios folder from your app if not do it and after do
````
npm run build
npx cap add ios
npx cap sync
npm run build
npx cap copy
npx cap open ios
```
and build your app
look at the [angular-sqlite-app-starter](https://github.com/jepiqueau/angular-sqlite-app-starter) to see all other changes you have to make in your code as since v2.9.16 quite changes and enhancements occur.

**Comment by michaelbutler1998** [comment]
@jepiqueau, thanks for taking the time to assist me here.

Just tried the advice, no luck. See below my package.json:

`{
  "name": "cardTwister",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "serve": "vue-cli-service serve",
    "build": "vue-cli-service build",
    "lint": "vue-cli-service lint"
  },
  "dependencies": {
    "@capacitor-community/admob": "^1.2.2",
    "@capacitor-community/firebase-analytics": "^0.3.3",
    "@capacitor-community/sqlite": "^2.9.13",
    "@capacitor/android": "^2.5.0",
    "@capacitor/cli": "^3.4.1",
    "@capacitor/core": "^3.4.1",
    "@capacitor/ios": "^3.4.1",
    "@ionic-native/core": "^5.23.0",
    "@ionic-native/google-analytics": "^4.20.0",
    "@ionic-native/in-app-purchase": "^5.24.0",
    "@ionic-native/in-app-purchase-2": "^5.36.0",
    "@ionic-native/keyboard": "^5.35.0",
    "@ionic-native/launch-review": "^5.24.0",
    "@ionic-native/onesignal": "^5.23.0",
    "@ionic-native/screen-orientation": "^5.19.1",
    "@ionic/core": "^4.11.7",
    "@ionic/vue": "0.0.9",
    "@sentry/browser": "^5.15.5",
    "@sentry/integrations": "^5.15.5",
    "apollo-boost": "^0.4.9",
    "capacitor-keep-screen-on": "^1.0.0-alpha.6",
    "cordova-launch-review": "^4.0.0",
    "cordova-plugin-device": "^2.0.3",
    "cordova-plugin-ionic-keyboard": "^2.2.0",
    "cordova-plugin-purchase": "^10.6.1",
    "cordova-plugin-screen-orientation": "^3.0.2",
    "core-js": "^3.4.4",
    "es6-promise-plugin": "^4.2.2",
    "graphql": "^15.5.0",
    "jetifier": "^1.6.6",
    "onesignal-cordova-plugin": "^2.11.1",
    "rxjs": "^7.5.4",
    "vue": "^2.6.10",
    "vue-apollo": "^3.0.7",
    "vue-router": "^3.1.3",
    "vue2-touch-events": "^3.2.2",
    "vuex": "^3.1.2"
  },
  "devDependencies": {
    "@vue/cli-plugin-babel": "^4.1.0",
    "@vue/cli-service": "^4.1.0",
    "node-sass": "^4.13.1",
    "node-sass-magic-importer": "^5.3.2",
    "sass-loader": "^8.0.2",
    "vue-sqlite-hook": "^2.0.0-alpha.1",
    "vue-template-compiler": "^2.6.10"
  },
  "eslintConfig": {
    "root": true,
    "env": {
      "node": true
    },
    "extends": [
      "plugin:vue/essential",
      "eslint:recommended"
    ],
    "rules": {},
    "parserOptions": {
      "parser": "babel-eslint"
    }
  },
  "browserslist": [
    "> 1%",
    "last 2 versions"
  ]
}
`

**Comment by jepiqueau** [comment]
@michaelbutler1998 it seems that you upgrade to capacitor 3 but did not upgrade @capacitor-community/sqlite you must install 3.4.0-1 and vue-sqlite-hook. Now you are still using vue 2 series and a  old @ionic/vue. this is to far away of what i am using so i cannot really help My advice will be to revamp your complete app by using all  the latest releases or keep on using the capacitor 2. There is no way that i can help here. i have some vue app starters using vue 3 on [https:/github.com](https://github.com/jepiqueau) which can help you starting

**Comment by jepiqueau** [comment]
@michaelbutler1998 any news?

**Comment by michaelbutler1998** [comment]
Hey @jepiqueau, apologies for the delayed response, Monday - Friday I have another full-time job and with this project, I have two large changes (one of them being this) that I'm working on simultaneously so have not had a chance to dive back into this one as of yet.

Please leave the thread open, I want to contribute and help others when I solve this issue so would like to be able to return to it and will provide feedback having tested the above suggestion as soon as possible

**Comment by michaelbutler1998** [comment]
Thank you @jepiqueau, your demo repos were incredibly useful.

To resolve the error, I did the following:
* removed all offending apps (this was not the only only displaying this issue) with npm unintall @...
* upgraded capacitor from 2.0 to 3.0 from scratch to check no steps were missed
* Reinstalled the apps one at a time, changing the imports from the capacitor 2 method to capacitor 3 method
I.e. import { CapacitorSQLite, SQLiteConnection } from '@capacitor-community/sqlite';

Cleared cache in xCode by pressing CMD+Shift+K

Ran the project and the issue did not reoccur.

Thanks again
